src/Application/Program.cs

```diff
@@ -45,6 +46,7 @@ static void Main(string[] args)
                     }
                 }
             }
+            form.Show();
```

Comment:
Actually it is wrong to move form.Show down here.  I want "form.Show" first before opening anything so that the form shows up ASAP before file load operation kicks in.

Reply:
it will be a frozen form that user can't do anything cause of your code.
this change will improve the performance

Reply:
If the "open" call hangs forever sure, but at least the user sees a window, as opposed to seeing nothing at all, then you don't know if the app even launched.  If the open call takes 1 second, then you see xmlnotepad quickly (subsecond) then the document populates.  That's the experience I'm shooting for.  I always like apps that launch as quickly as possible.

Reply:
Even when it is not working and everybody things that it crashed?
I think we can work more on loading performance

Reply:
I feel strongly about this one, sorry.